styles.css

```diff
@@ -0,0 +1,155 @@
+/*
+* Prefixed by https://autoprefixer.github.io
+* PostCSS: v8.4.14,
+* Autoprefixer: v10.4.7
+* Browsers: last 4 version
+*/
+
+body {
```

Comment:
- [ ] Your Project looks Good but I just keep encouraging you to follow the guidelines for Design for this project, I noticed that you forget to add the `Background` Icones which are on the Figma template, since we are imitating the Figma Template you should add those Icons and fixed them the way they look like on the original design  , kindly fix this :wink:  :point_down: 

Figma Template 
![Screenshot from 2023-02-17 16-35-03](https://user-images.githubusercontent.com/88894525/219684565-649980aa-4f60-4652-85e2-6433030c91bc.png)